EXAMPLES.md

```diff
@@ -0,0 +1,60 @@
+# PromQL usage examples for Fortigate Exporter
+
+PromQL is an advanced language and proper usage of it requires one to both understand
+the language itself and how the metrics one has are constructed.
+
+To help all the users of this exporter, here are some examples to illustrate
+how one could use the power of PromQL to query interesting facts about one's
+environment.
```

Comment:
Maybe we should here a link to the PromQL documentation?   https://prometheus.io/docs/prometheus/latest/querying/examples/